managed/src/main/resources/reference.conf

```diff
@@ -118,6 +120,7 @@ runtime_config {
       "yb.taskGC."
       "yb.alert.max_clock_skew_ms"
       "yb.proxy_endpoint_timeout"
+      "yb.customer_task_db_query_limit"
```

Comment:
I wonder who backported this list. I do not think there is anything using it. 
Can you check any usages of  runtime_config in 2.4? In mainline it is line 80 in RuntimeConfigController. In 2.4 we should not even have a controller since there is no "real" runtime config.

Reply:
Okay I checked that there is no RuntimeConfigController in 2.4.
Just remove this whole config starting from line 117 onwards.

Reply:
I though that this is 2.4 backport. So ignore my comments.